fvtest/porttest/CMakeLists.txt

```diff
@@ -75,30 +47,56 @@ omr_add_executable(omrporttest
 	si_numcpusTest.cpp
 	testHelpers.cpp
 	testProcessHelpers.cpp
-
-	${cudaSources}
-	${socketSources}
 )
 
+if(OMR_OPT_CUDA)
+	target_sources(omrporttest
+		PRIVATE
+			cudaBasic.cpp
+			cudaError.cpp
+			cudaEvent.cpp
+			cudaInvalid.cpp
+			cudaLaunch.cpp
+			cudaLink.cpp
+			cudaMemory.cpp
+			cudaModule.cpp
+			cudaPeer.cpp
+			cudaProperty.cpp
+			cudaPtx.cpp
+			cudaStream.cpp
+			cudaTests.cpp
+	)
+endif()
 
+# TODO: Remove if(NOT OMR_HOST_OS STREQUAL "win") after OMRSOCK API is implemented on Windows.
+if(NOT OMR_HOST_OS STREQUAL "win")
+	target_sources(omrporttest
+		PRIVATE
+			omrsockTest.cpp
+	)
+endif()
 
 target_link_libraries(omrporttest
 	omrGtestGlue
 	omrtestutil
 	${OMR_PORT_LIB}
 )
+
 if(OMR_HOST_OS STREQUAL "win")
 	target_link_libraries(omrporttest
 		ws2_32
 	)
 endif()
 
 if(OMR_HOST_OS STREQUAL "zos")
-	target_link_libraries(omrporttest j9a2e)
+	target_link_libraries(omrporttest 
+		j9a2e
+	)
 endif()
 
-omr_add_library(sltestlib SHARED
-	sltestlib/sltest.c
+omr_add_library(sltestlib 
```

Comment:
Same comment as aixbaddep, in this case the SHARED flag actually does belong up on the first line, and this formatting was OK.